### PR TITLE
Minor Fixes in Website

### PR DESCRIPTION
**Pull request by ayushyamitabh**
### Info
**Time Spent:** 2 hours

### Changes
```
website/.stylelintignore
```
- Added `*.svg` to ignore our logo file

----

```
website/public/index.html
```
- Change the favicon route to match new logo

----

```
website/public/logo.svg | website/src/resources/logo.svg
```
- New logo, `.svg` should be changed to `.ico` later

---- 
```
DashboardHeader.jsx
```
- Better Icon handling 

----
```
ProfileInfo.jsx | ProfileInfo.css
```
- Remove Profile Picture
- Add Logo
- Add Logo click to go back to dashboard

----
```
DashboardRouter.jsx
```
- Pass down userData as props
- Improve API call syntax 

----
```
Profile.jsx | Profile.css
```
- Center loading icon
- Get User Data from props instead of duplicate API call

### CODE REVIEW

File: website/.stylelintignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /src/Google-Sans
 *.jsx
+*.svg
```
Comment (whuang001) [inline website/.stylelintignore]: add a new line here

Comment (khelif96) [inline website/.stylelintignore]: New line

File: website/public/logo.svg [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (khelif96) [inline website/public/logo.svg]: Maybe we could move this to a `res/` folder with all the other images

File: website/src/components/DashboardHeader/DashboardHeader.jsx [hunk 3]
```diff
@@ -44,15 +52,14 @@ class DashboardHeader extends PureComponent {
       },
     ];
     if (teacher) {
-      Icons.unshift({
-        type: 'add_circle',
-        onClick: () => { window.location = '/create/class'; },
-        key: 'add_circle',
-      });
+      Icons.unshift(...TeacherIcons);
```
Comment (khelif96) [inline website/src/components/DashboardHeader/DashboardHeader.jsx]: Nice

File: website/src/components/DashboardHeader/DashboardHeader.jsx [hunk 4]
```diff
@@ -44,15 +52,14 @@ class DashboardHeader extends PureComponent {
...
     }
     return (
       <Header className="Header">
-        <ProfileInfo name={firebase.auth().currentUser ? firebase.auth().currentUser.displayName : 'User Name'} email={firebase.auth().currentUser ? firebase.auth().currentUser.email : 'email@domain.com'} />
+        <ProfileInfo
+          name={firebase.auth().currentUser ? firebase.auth().currentUser.displayName : 'User Name'}
```
Comment (khelif96) [inline website/src/components/DashboardHeader/DashboardHeader.jsx]: I know this wasnt you but we can probably put this into a variable `let currentUser = firebase.auth().currentUser` and then do this `currentUser ? currentUser.displayName ... `

File: website/public/logo.svg (hunk 2 above)
Comment (ayushyamitabh) [reply website/public/logo.svg]: the favicon should be in the public folder -- read this in some documentation for react, don't remember where

File: website/src/components/DashboardHeader/DashboardHeader.jsx (hunk 4 above)
Comment (ayushyamitabh) [reply website/src/components/DashboardHeader/DashboardHeader.jsx]: aii ye

**Pull request merged**